priv/repo/seeds.exs

```diff
@@ -12,4 +12,4 @@
 EChronicler.Repo.insert!(%EChronicler.Models.JournalEntry{title: "Awesome Blog Post", author: "Grace", entry: "Time is an illustion, teatime doubly so."})
 EChronicler.Repo.insert!(%EChronicler.Models.JournalEntry{title: "Cool Blog Post", author: "Grace", entry: "Test."})
 EChronicler.Repo.insert!(%EChronicler.Models.JournalEntry{title: "So-So Blog Post", author: "Grace", entry: "Don't panic."})
-EChronicler.Repo.insert!(%EChronicler.Models.JournalEntry{title: "Regular Blog Title", author: "Normal Name", entry: "Words."})
+EChronicler.Repo.insert!(%EChronicler.Models.JournalEntry{title: "Regular Blog Title", author: "Normal Name", entry: "Words words words. Words words words.Words words words.Words words words.Words words words.Words words words.Words words words. Words words words.Words words words.Words words words.Words words words.Words words words.Words words words.Words words words.Words words words.Words words words.Words words words.Words words words.Words words words.Words words words.Words words words.Words words words.Words words words."})
```

Comment:
It's useful for your seed data to reflect what actual user data is going to look like, particularly since that will be what you demo off of. Thinking on your own blog posts, they're much longer than even this "long" example here. Try putting in some lorem ipsum text with paragraph breaks.

Reply:
That first `lib/eChronicler_web/templates/journal_entry/index.html.heex` was an extra file that snuck in with my git rebase that I didn't want to have... Oops, I apologize for that. The two templates in `e_chronicler_web/templates/journal_entry` should all be good and include the syntactic HTML tags

Reply:
I added something with line breaks and realized HTML didn't render them automatically so I added a new tag with `white-space: pre-line;` in the css and it seems to work well!